src/api/TokenIOSync.h

```diff
@@ -122,15 +123,15 @@
              authorization:(BankAuthorization *)authorization;
 
 /**
- * Sends a notification to request adding of a key
+ * Sends a notification to request adding keys
  *
  * @param alias alias to notify
- * @param key key in string form
- * @param keyName optional name of key
+ * @param keys list of new keys to add
+ * @param deviceMetadata device metadata of the keys
```

Comment:
does this info get shown to user when prompting them? if so, good to mention that.